Python/Product/VSInterpreters/PackageManager/PipPackageCache.cs

```diff
@@ -160,7 +161,7 @@ string cachePath
                     }
                 }
 
-                return _cache.Values.Select(p => p.Clone()).ToList();
+                return _cache.Values.ToList().AsReadOnly();
```

Comment:
slight change. I dont believe we need to clone. since 'ToList' is already making a copy.

tested in a playground
https://dotnetfiddle.net/Widget/6U9bNG

Reply:
Does ReadfOnlyCollection not implement IList? Wondering why the return type needs changing here.

Reply:
Ok. I think I can remove all these interface changes because in the end I'm still calling ToList(), which makes a copy anyways.

Reply:
Maybe in the future this should be lazy and just be IEnumerable.

Reply:
removed